functional/webhook-certificate-on-localhost/test.sh

```diff
@@ -0,0 +1,229 @@
+        rlRun "limeUpdateConf verifier server_key verifier-key.pem"
+        rlRun "limeUpdateConf verifier client_cert ${CERTDIR}/verifier-client-cert.pem"
+        rlRun "limeUpdateConf verifier client_key ${CERTDIR}/verifier-client-key.pem"
+        rlRun "limeUpdateConf revocations enabled_revocation_notifications '[\"agent\", \"webhook\"]'"
+        rlRun "limeUpdateConf agent enable_revocation_notifications true"
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        rlRun "limeUpdateConf tenant tls_dir $CERTDIR"
+        rlRun "limeUpdateConf tenant trusted_server_ca '[\"intermediate-cacert.pem\", \"good-cacert.pem\"]'"
+        rlRun "limeUpdateConf tenant client_cert tenant-cert.pem"
+        rlRun "limeUpdateConf tenant client_key tenant-key.pem"
+        # registrar
+        rlRun "limeUpdateConf registrar check_client_cert True"
+        rlRun "limeUpdateConf registrar tls_dir $CERTDIR"
+        rlRun "limeUpdateConf registrar trusted_client_ca '[\"intermediate-cacert.pem\", \"good-cacert.pem\"]'"
+        rlRun "limeUpdateConf registrar server_cert registrar-cert.pem"
+        rlRun "limeUpdateConf registrar server_key registrar-key.pem"
+        # agent
+        rlRun "limeUpdateConf agent trusted_client_ca '\"['${CERTDIR}/intermediate-cacert.pem', '${CERTDIR}/good-cacert.pem']\"'"
+        rlRun "limeUpdateConf agent server_key '\"agent-key.pem\"'"
+        rlRun "limeUpdateConf agent server_cert '\"agent-cert.pem\"'"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        rlRun "limeStartAgent"
+        rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
+        # create allowlist and excludelist
+        limeCreateTestPolicy
+    rlPhaseEnd
+
+    for i in "good-webhook good 8980 none" "bad-webhook bad 8981 CERTIFICATE_VERIFY_FAILED" "none bad 8982 SSLError" "installed good 8993 none"; do
+        read -r WEBHOOK EXPECTED_RESULT WEBHOOK_SERVER_PORT EXPECTED_ERROR <<< "${i}"
+
+        # This case is to test that the verifier is including the system-wide
+        # installed certificates when verifying the revocation notification
+        # webhook certificate
+        if [ "${WEBHOOK}" = "installed" ]; then
+            rlPhaseStartTest "Install CA certificate on system-wide store, and run on port ${WEBHOOK_SERVER_PORT}"
+                rlRun "limeUpdateConf revocations webhook_url https://localhost:${WEBHOOK_SERVER_PORT}"
+                # Install the "bad" CA certificate on system-wide trust store
+                rlRun "cp $(x509Cert bad-ca) /etc/pki/ca-trust/source/anchors/keylime-ca.crt"
```

Comment:
could you rename `keylime-ca.crt` to `webhook-ca.crt`? I find keylime a bit confusing here.